src/Data/Yaml.php

```diff
@@ -50,8 +50,13 @@ public static function decode($string): array
 		};
 	}
 
+	/**
+	 * Returns which YAML parser (`spyc` or `symfony`)
+	 * is configured to be used
+	 * @internal
+	 */
 	public static function handler(): string
 	{
-		return App::instance(null, true)?->option('yaml') ?? 'spyc';
+		return App::instance(null, true)?->option('yaml.handler') ?? 'spyc';
```

Comment:
```suggestion
		return App::instance(null, true)?->option('yaml.handler', 'spyc');
```

Reply:
This won't work if the lazy App::instance returns null

Reply:
Oh sorry, I missed that!